.travis.yml

```
@@ -42,3 +42,13 @@ deploy:
     on:
       branch: "all-pgautofailover"
       condition: -d "pkgs/releases"
+  - provider: packagecloud
+    username: "citusdata"
+    repository: "$PKG_REPOTYPE-nightlies"
+    token: $PACKAGECLOUD_API_TOKEN
+    dist: $TARGET_PLATFORM
+    skip_cleanup: true
+    local-dir: "pkgs/nightlies"
+    on:
+      branch: "nightly_test"
```

```suggestion
      branch: "all-pgautofilover"
```

We should do this change after we test the packages.